deploy/olm-catalog/bundle/manifests/oadp-operator.v0.2.3.clusterserviceversion.yaml

```diff
@@ -213,6 +213,10 @@ spec:
                   value: konveyor
                 - name: VELERO_REPO
                   value: velero
+                - name: VELERO_REGISTRY_REPO
```

Comment:
Should we add these vars in `operator.yaml` as well for supporting non-OLM installation ?